donut/modules/account/helpers.py

```diff
@@ -91,13 +95,15 @@ def handle_request_account(uid, last_name):
     Tuple (bool, string). The bool indicates success. If not successful,
     the string is an error message.
   """
-    query = sqlalchemy.text("""
+    s = """
     SELECT first_name, last_name, email, username
     FROM members
       NATURAL LEFT JOIN users
-    WHERE uid = :uid
-    """)
-    result = flask.g.db.execute(query, uid=uid).first()
+    WHERE uid = %s
+    """
+    with flask.g.pymysql_db.cursor() as cursor:
+        cursor.execute(s, [uid])
+        result = cursor.fetchone()
     if result is None or result["last_name"].lower() != last_name.lower():
```

Comment:
Should you check that the keys exist? Or is it guaranteed.

Reply:
guaranteed since we selected them